### PR TITLE
Fix tasks

### PR DESCRIPTION
**Pull request by ehoelzl**
Fixes the Epsilon and CIFAR10 tasks

### CODE REVIEW

File: pytorch/imagerecognition/cifar10-resnet20-all-reduce-scaling/main.py [hunk 1]
```diff
@@ -60,14 +57,18 @@ def train_loop(
     rank = dist.get_rank()
     world_size = dist.get_world_size()
 
+    # LR = 0.1 / 256 / sample
```
Comment (Panaetius) [inline pytorch/imagerecognition/cifar10-resnet20-all-reduce-scaling/main.py:65]: is this commented code needed?

**Review by Panaetius — CHANGES_REQUESTED** [review]
Looks great, thanks. One minor cleanup issue

File: pytorch/imagerecognition/cifar10-resnet20-all-reduce-scaling/main.py (hunk 1 above)
Comment (ehoelzl) [reply pytorch/imagerecognition/cifar10-resnet20-all-reduce-scaling/main.py:65]: Yes, its just to explain where the formula comes from for the LR

**Review by Panaetius — APPROVED** [review]
Thank you

**Pull request merged**